src/herder/test/HerderTests.cpp

```diff
@@ -3960,6 +3962,176 @@ TEST_CASE("do not flood invalid transactions", "[herder]")
     REQUIRE(txSet->checkValid(*app, 0, 0));
 }
 
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
```

Comment:
Currently adding another test here that mixes Soroban and classic txs in the same ledger (to ensure two queues are working as expected)